### PR TITLE
Bug 1684134 add multi-stage builds TP line to Release Notes for 3.11

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1684134

### CODE REVIEW

File: release_notes/ocp_3_11_release_notes.adoc [hunk 1]
```diff
@@ -1548,6 +1548,11 @@ features marked *GA* indicate _General Availability_.
 |-
 |TP
 
+|Multi-stage builds in Dockerfiles managed by the S2I image builder
```
Comment (xiuwang) [inline release_notes/ocp_3_11_release_notes.adoc]: I suspect ‘S2I image builder’ here if correct, most display 'image builder'.
See https://github.com/openshift/imagebuilder#oci-image-builder 
S2I means source-to-image.

**Comment by bmcelvee** [comment]
Thanks, @xiuwang! Updated the PR and going ahead merging. 

Minor change - no peer review required.

**Comment by xiuwang** [comment]
@bmcelvee Thanks～

**Pull request merged**